lib/gauge_fix_ovr.cu

```diff
@@ -71,7 +71,10 @@ namespace quda {
     bool advanceAux(TuneParam &param) const
     {
       param.aux.x = (param.aux.x + 1) % 6;
-       // mu must be contained in the block, types 0, 1, 2 have mu = 8 and 3, 4, 5 have mu = 4
+      if (!device::shared_memory_atomic_supported()) { // 1, 4 use shared memory atomics
```

Comment:
Is lack of shared memory atomics a thing on some SYCL platforms?

Reply:
This is mainly for testing.  I don't think there is actually a way to query it, so is seems to expect it to be supported.